thirdparty/openapi3/schemas/GenerateChallengeRequest.yaml

```diff
@@ -0,0 +1,10 @@
+title: GenerateChallengeRequest
```

Comment:
Should this be be `ConsentGenerateChallengePostRequest`?

Reply:
Seems that it should be ConsentGenerateChallengeIDPostRequest. That should be good enough for now.